test/WebSites/FSharpWebSite/FSharpWebSite.fsproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="FSharp.NET.Sdk; Microsoft.NET.Sdk.Web">
+  <Import Project="..\..\..\build\dependencies.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net46;netcoreapp2.0</TargetFrameworks>
```

Comment:
Remove `net46` from here and use `<TargetFramework>` (singular) tag instead. See my email from yesterday for info.